### PR TITLE
Added Appear modules

### PR DESCRIPTION
**Pull request by kennethdammyr**

### CODE REVIEW

File: module-types/Appear/ECx110.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+manufacturer: Appear
```
Comment (sjurtf) [inline module-types/Appear/ECx110.yaml:2]: Previously we have named this manufacturer `AppearTV`.
The modules should match.

We must either change the name of these modules to `AppearTV`, or the name of the manufacturer of the devices to `Appear`.

File: module-types/Appear/ECx110.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  - name: 2A
+    type: BNC
+    rear_port: 'Backplane interconnect'
+    rear_port_position: 5
+  - name: 2B
+    type: BNC
+    rear_port: 'Backplane interconnect'
+    rear_port_position: 6
+  - name: 2C
+    type: BNC
```
Comment (sjurtf) [inline module-types/Appear/ECx110.yaml:35]: `BNC` should be lowercase.

File: module-types/Appear/IPx210.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+manufacturer: Appear
+model: IPx110
```
Comment (sjurtf) [inline module-types/Appear/IPx210.yaml:3]: This should be `IPx210`

File: module-types/Appear/ECx110.yaml (hunk 1 above)
Comment (kennethdammyr) [reply module-types/Appear/ECx110.yaml:2]: I know, but we should change to only "Appear". I believe the company has been rebranded...

File: module-types/Appear/IPx210.yaml (hunk 3 above)
Comment (kennethdammyr) [reply module-types/Appear/IPx210.yaml:3]: Flaut.

**Review by sjurtf — APPROVED** [review]

**Pull request merged**